core/src/main/java/be/cytomine/controller/ontology/RestUserAnnotationController.java

```diff
@@ -109,7 +109,9 @@ public void downloadDocumentByProject(
     ) throws IOException {
         Project project = projectService.find(idProject)
                 .orElseThrow(() -> new ObjectNotFoundException("Project", idProject));
-        users = userService.fillEmptyUserIds(users, idProject);
```

Comment:
can you use a `Optional<String> users` here ?  So you don't need to test for null after